ci-operator/step-registry/cucushift/installer/check/vsphere/multi-vcenter/cucushift-installer-check-vsphere-multi-vcenter-commands.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+export KUBECONFIG=${SHARED_DIR}/kubeconfig
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/govc.sh"
+# shellcheck source=/dev/null
+source "${SHARED_DIR}/vsphere_context.sh"
+# These two environment variables are coming from vsphere_context.sh and
+# the file they are assigned to is not available in this step.
+unset SSL_CERT_FILE
+unset GOVC_TLS_CA_CERTS
+INSTALL_CONFIG="${SHARED_DIR}/install-config.yaml"
+
+check_result=0
+infra_id=$(oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster)
+
+function check_multi_vcenter() {
+    local node_type=$1
+    local check_result=0 nodes_list node_region node_zone fd_folder fd_datacenter fd_server fd_servers
+    nodes_list=$(oc get nodes --selector node.openshift.io/os_id=rhcos,node-role.kubernetes.io/${node_type} -o json | jq -r '.items[].metadata.name')
+    readarray -t fd_servers < <(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains[*].server")
+    for node in ${nodes_list}; do
+        # shellcheck disable=SC2207
+        label_info=($(oc get node ${node} --show-labels --no-headers | awk -F, '{print $(NF-1),$NF}'))
+        node_region=$(echo ${label_info[0]} | awk -F= '{print $2}')
+        node_zone=$(echo ${label_info[1]} | awk -F= '{print $2}')
+        echo "checking labels on node ${node}, label region: ${node_region}, label zone: ${node_zone}"
+        fd_datacenter=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(zone==${node_zone}).topology.datacenter")
+        [[ -z "${fd_datacenter}" ]] && fd_datacenter=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.datacenter")
+	fd_folder=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(zone==${node_zone}).topology.folder")
+	fd_server=$(yq-go r ${INSTALL_CONFIG} "platform.vsphere.failureDomains.(zone==${node_zone}).server")
+        [[ -z "${fd_folder}" ]] && fd_folder="/${fd_datacenter}/vm/${infra_id}"	
+	if [[ -n "$(govc ls ${fd_folder}/${node})"  ]];then
```

Comment:
From tested job log for node master-0 checking:
```
+ for node in ${nodes_list}
+ label_info=($(oc get node ${node} --show-labels --no-headers | awk -F, '{print $(NF-1),$NF}'))
++ oc get node ci-op-wcinz0ih-91c3a-ttpq9-master-0 --show-labels --no-headers
++ awk -F, '{print $(NF-1),$NF}'
++ echo topology.kubernetes.io/region=us-central
++ awk -F= '{print $2}'
+ node_region=us-central
++ echo topology.kubernetes.io/zone=us-central-1a
++ awk -F= '{print $2}'
+ node_zone=us-central-1a
+ echo 'checking labels on node ci-op-wcinz0ih-91c3a-ttpq9-master-0, label region: us-central, label zone: us-central-1a'
checking labels on node ci-op-wcinz0ih-91c3a-ttpq9-master-0, label region: us-central, label zone: us-central-1a
++ yq-go r /tmp/secret/install-config.yaml 'platform.vsphere.failureDomains.(zone==us-central-1a).topology.datacenter'
+ fd_datacenter=cidatacenter-2
+ [[ -z cidatacenter-2 ]]
++ yq-go r /tmp/secret/install-config.yaml 'platform.vsphere.failureDomains.(zone==us-central-1a).topology.folder'
+ fd_folder=
++ yq-go r /tmp/secret/install-config.yaml 'platform.vsphere.failureDomains.(zone==us-central-1a).server'
+ fd_server=vcenter-1.ci.ibmc.devcluster.openshift.com
+ [[ -z '' ]]
+ fd_folder=/cidatacenter-2/vm/ci-op-wcinz0ih-91c3a-ttpq9
++ govc ls /cidatacenter-2/vm/ci-op-wcinz0ih-91c3a-ttpq9/ci-op-wcinz0ih-91c3a-ttpq9-master-0
+ [[ -n '' ]]
++ oc get machines.machine.openshift.io ci-op-wcinz0ih-91c3a-ttpq9-master-0 -n openshift-machine-api -ojson
++ grep server
++ awk -F ': ' '{print $2}'
++ sed 's/\"//g'
+ machine_server=vcenter-1.ci.ibmc.devcluster.openshift.com
+ [[ vcenter-1.ci.ibmc.devcluster.openshift.com == \v\c\e\n\t\e\r\-\1\.\c\i\.\i\b\m\c\.\d\e\v\c\l\u\s\t\e\r\.\o\p\e\n\s\h\i\f\t\.\c\o\m ]]
+ [[ 3 -gt 1 ]]
+ echo 'the vcenter specified in install-config is same with machine'
the vcenter specified in install-config is same with machine
```
As govc login ENV (line9) is for `vcenter`, while here you check machine on `vcenter-1`, so the command `govc ls` is always empty, we don't have any failure process here, so the check is passed, but the result is inaccurate.

BTW, when enabling debugging with command "set -x", please don't add it before line9 and line11 which contains sensitive credentials.